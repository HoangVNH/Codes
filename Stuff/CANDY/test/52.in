10
99
VVXXXVVXDVVDXXXVDXDXXXXDVDXDDXXVVXVVXXDDXVVDDVVDXXVVXXDDDXXXDXDXXXVXXVVXDVVVVVVXDVXXDVVDDXDXDDXDDXD
80
DXDDDXDDVVXVVDVXVXVVXVDDVXXDVDXDDDDVXXXVXDDDXXVVXXDDXVVXDXXXDXVDXXXDVVXVDXDVVDVV
64
VDDVVXDXVXXXXDVVVVXVDXVXVDXXXVVDDXXXVDDVVDDXDVVDXXDVVXDXVDDDXDVV
97
VDXXDXDXXDDXVVXVVXDXXXVDXVDDDVXXDVVXVXVDVXVVXVXXDVVDXXDDVVXVXVDXVVVXXVVXDDXXVXVVVXDVDDXVVDDVVXXDX
92
VXXVVDDDVVDVXVDXDDXXVXXDVDDDXXXXDXXVVVVVVDXDXVXXXDDDDXVXVXVVXXXDVXXVXVXVXDDDVDXVDXDDVVXVXXXV
85
XDDDXVVXXDXDDXDXXDVVVDDXVVVDXVXVVDXXXVDXXDDDDXDVDXXDVDDXDVDDXVXVDXVVXVVDDDDVXVXDDVVVX
70
DDXVDDVXXDDXXXVXVDDDDVDXXDXDVDVXXXVXDDXXDVVXVVXDVDXDDDDDVVXXDVXXVXXVDV
88
DDDDVVXDDVDDVXVDDXDDDXVXDDVXDVVXDDXVXDVXVDVVXDDXXXVDXDXDDXVXXXVDXVDDXXDXVVXDVVDVVVXDDVXV
59
XDVVVXXVXXDVXDDVVDVVXXDXDXXDDVDVDVVXDXVDVXDXVVXXXDVDXDDDDXD
61
XDVDXXVDVDXVVDXDVDDVXDDXDXVDXDXVDXDVXVXXDXVDXVDDDVXXVXXDDXVDV
