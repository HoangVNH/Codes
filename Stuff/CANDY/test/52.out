98
79
63
96
90
84
69
87
58
59
